woocommerce/woocommerce-android
# Fix download path for Play Store metadata translations from GlotPress

### Description

I broke it in [`b45151c`](https://github.com/woocommerce/woocommerce-android/commit/b45151c2e593419f9f8742639f12b2a6b808788a#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688L506-R505) with a too-eager DRY refactor and it resulted in Play Store metadata being incorrectly downloaded to `WooCommerce/metadata/` instead of the usual `fastlane/metadata/android` in [`a6d4981` (#6813)](https://github.com/woocommerce/woocommerce-android/pull/6813/commits/a6d49813f1d1138c8dcd311a947e4a93e87931d1)

### Testing instructions
The change is pretty straightforward and we can use the [diff of the commit that broke it](https://github.com/woocommerce/woocommerce-android/commit/b45151c2e593419f9f8742639f12b2a6b808788a#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688L506-R505) as well as [the code in WordPress Android](https://github.com/wordpress-mobile/WordPress-Android/blob/658b7c94b14b525f9dffae45fd258fd1f2853a43/fastlane/lanes/localization.rb#L222) (with `app_values[:metadata_dir]` is defined [here](https://github.com/wordpress-mobile/WordPress-Android/blob/658b7c94b14b525f9dffae45fd258fd1f2853a43/fastlane/Fastfile#L3-L16)) as a comparison.

Observing the shape of the diff in this PR also serves as a confirmation of the fix legitimacy.

Also note that  [`f06efd4`](https://github.com/woocommerce/woocommerce-android/pull/6814/commits/f06efd4ae9743d0d060c5775ab5ff630187429aa) is the result of running the updated automation, via `bundle exec fastlane download_metadata_strings name:9.4 build_number:309` with a then amended commit message (`git commit --amend`).

---
